src/apiServices/__isomorphic-AuthService.ts

```diff
@@ -0,0 +1,143 @@
+// // PLEASE NOTE THIS FILE IS NOT CURRENTLY IN USE AND WILL BE REFACTORED OR REMOVED BEFORE OFFICIAL RELEASE
```

Comment:
We can probably delete this file now - we have version control if we need to recover it at any point.